### ISSUE TITLE
Contributing process

### ISSUE THREAD

**Issue by LegitTalon** [nodeschool/denver-boulder#9]
It would be nice to document how someone can contribute to the website.

**Comment by ralucas** [comment]
Agreed.  Is there a style guide that you've established for the site?

**Comment by LegitTalon** [comment]
Not yet? I suppose that should be another issue
